jax/_src/interpreters/mlir.py

```diff
@@ -75,6 +75,12 @@
          "Supports the special value 'sponge' to pick the path from the "
          "environment variable TEST_UNDECLARED_OUTPUTS_DIR.")
 
+_JAX_ENABLE_IR_DEBUG = config.DEFINE_string(
+  'jax_enable_ir_debug', os.getenv('JAX_ENABLE_IR_DEBUG', "True"),
```

Comment:
Perhaps `jax_include_debug_information_in_dumps` ?

It isn't immediately clear to me what `enable_ir_debug` means.